src/components/share-url.ts

```diff
@@ -0,0 +1,44 @@
+import { LitElement, css, html } from "lit";
+import { customElement } from "lit/decorators.js";
+import { ICON_CHECK_SLOTTED, ICON_COPY_SLOTTED } from "../const";
+
+@customElement("share-url")
+export class ShareURL extends LitElement {
+  private copied: boolean = false;
+
+  render() {
+    return html`
+      <md-text-button
+        trailing-icon
+        @click=${this.copyToClipboard}
+        ?disabled=${this.copied}
+      >
+        ${this.copied ? "URL copied to clipboard" : "Copy URL to clipboard"}
+        ${this.copied ? ICON_CHECK_SLOTTED : ICON_COPY_SLOTTED}
+      </md-text-button>
+    `;
+  }
+
+  async copyToClipboard() {
+    if (this.copied) return;
+
+    await navigator.clipboard.writeText(
+      "https://ohf-voice.github.io/wake-word-collective/",
+    );
+
+    this.copied = true;
+    this.requestUpdate("copied");
+
+    setTimeout(() => {
+      this.copied = false;
+      this.requestUpdate("copied");
+    }, 5000);
+  }
+
+  static styles = css`
+    md-text-button {
+      margin-top: 16px;
```

Comment:
Margins should never be part of a component itself. It should always be applied by the component that includes them. This improves reusability.